src/components/WithInteractionManaged/index.js

```diff
@@ -37,7 +37,7 @@ export function withInteractionsManagedNoStatusBar(Component, Placeholder) {
         }
         style={{flex: 1}}
         ref={transitionRef}>
-        <StatusBar translucent={false} />
+        <StatusBar translucent={false} barStyle={'light-content'} />
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `StatusBar` component has been updated to set the `barStyle` prop to `'light-content'`. This change will adjust the appearance of the status bar content to light, which is suitable for dark backgrounds. However, please ensure that this change doesn't affect readability on screens with light backgrounds.
```diff
- <StatusBar translucent={false} />
+ <StatusBar translucent={false} barStyle={'light-content'} />
```
If there are screens with light backgrounds, consider passing the `barStyle` as a prop to the `WithInteractionManaged` component and use it here. This way, you can control the `barStyle` based on the background of the screen where this component is used.


<!-- This is an auto-generated comment by OSS CodeRabbit -->